main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+from json import load
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json") as file:
+        players_dict = load(file)
+
+    for player_name, info in players_dict.items():
+
+        race_info = info["race"]
+
+        if not Race.objects.filter(
+                name=race_info["name"]
+        ).exists():
+            Race.objects.create(
+                name=race_info["name"],
+                description=race_info["description"],
+            )
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create